Bootstrap/BootstrapTests/DynamicTypeTests.swift

```diff
@@ -0,0 +1,62 @@
+import Quick
+import Nimble
+import Nimble_Snapshots
+import Bootstrap
+
+class DynamicTypeTests: QuickSpec {
+    override func spec() {
+        describe("in some context", {
+            var view: UIView!
+
+            beforeEach {
+                setNimbleTolerance(0)
+                setNimbleTestFolder("tests")
+                view = DynamicTypeView(frame: CGRect(x: 0, y: 0, width: 500, height: 500))
+            }
+
+            it("has a valid snapshot for all content size categories (iOS 9)") {
```

Comment:
I know I could use device agnostic screenshots here, but I didn't want to test that (yet)